source/ShipJumpNavigation.cpp

```diff
@@ -128,18 +125,14 @@ double ShipJumpNavigation::JumpDriveFuel(double distance) const
 // If no jump method is possible, returns JumpType::None with a jump cost of 0.
 pair<JumpType, double> ShipJumpNavigation::GetCheapestJumpType(const System *destination) const
 {
-	if(!ship->GetSystem())
-		return make_pair(JumpType::NONE, 0.);
-	return GetCheapestJumpType(ship->GetSystem(), destination);
+	return GetCheapestJumpType(currentSystem, destination);
 }
 
 
 
 // Get the cheapest jump method between the two given systems.
 pair<JumpType, double> ShipJumpNavigation::GetCheapestJumpType(const System *from, const System *to) const
 {
-	if(!from || !to)
-		return make_pair(JumpType::NONE, 0.);
 	bool linked = from->Links().count(to);
```

Comment:
this too?

Reply:
This one was a safety tbf I dont see why not keep it
The other one just calls this function so no need to check twice but here it can prevent a crash

That said we prob should allow a crash rsther than weird behavior so we can remove it